orchestra/templates/communication/available_staffing_requests.html

```diff
@@ -24,7 +24,15 @@
         {% for request in requests %}
         <tr>
           <td>{{ request.workflow_description }}</td>
-          <td>{{ request.project_description }}</td>
+          <td class="tags-col">
+            {{ request.project_description }}
+            <br>
+            {% for tag in request.tags %}
+              <span class="label label-{{tag.status}}">
+                {{tag.label}}
+              </span>
+            {% endfor %}
+          </td>
           <td>{% if request.role_counter > 0 %}[Review] {% endif %}{{ request.step_description }}</td>
           <td>{{ request.detailed_description | safe }}</td>
```

Comment:
QA: do tags on the dashboard still work?

Reply:
They do! I checked! I forgot to send the screenshot for that!

Reply:
Can you make sure they still do after running both `yarn dev` and `gulp`?

Reply:
It disappeared after I ran 'gulp'! I updated the scss file. Now it works after running both!